src/loginServices.js

```diff
@@ -182,8 +182,14 @@ function dispatchLocale(url, store, tenant) {
  * @returns {Promise}
  */
 export function getLocale(okapiUrl, store, tenant) {
+  const query = [
+    'module==ORG',
+    'configName == localeSettings',
+    '(cql.allRecords=1 NOT userId="" NOT code="")'
```

Comment:
Please tell me `userId=""` is not how RMB does a "this field is not present" seach?

Reply:
Alas, `cql.allRecords=1 NOT userId=""` is _exactly_ how CQL specifies [all records where `userId` is not defined](https://github.com/folio-org/raml-module-builder/tree/57ad315376b1bb9e9b6f7d25e89c31ebcfdd4635#cql-matching-undefined-or-empty-values). Given that ui-tenant-settings [only provides `module` and `configName` when storing settings](https://github.com/folio-org/ui-tenant-settings/blob/c1c5514f444092c82867db6f4392069444746219/src/settings/Locale.js#L40-L41), asking for `userId` and `code` to be absent feels like the correct way to retrieve a unique record.

Reply:
Well, that (`NOT userId=""`) is horrifying.

BTW., in CQL, `foo and (cql.allRecords=1 not bar)` is exactly equivalent to (foo not bar)`. I _would_ suggest using the more concise version of the syntax here, but since this area already reeks of special-cases, I wouldn't be totally confident that the involvement of `cql.allRecords=1` isn't also special here. So if you've tested the present version and it does what you want, I say leave it.

Reply:
I got my sample from the RMB docs so I'm gonna stick with it, since I think most folks on FOLIO will use that as a reference. But I appreciate you teaching me about the other options!